### PR TITLE
Functional Tests : BO - Advanced Parameter - Authorization Server - View detail

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : BO - Advanced Parameter - Authorization Server - View detail
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/5656655068
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    describe('View the application', async () => {
+      it('should go to the details page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToDetailsPage', baseContext);
+
+        await authorizationServerPage.goToViewAuthorizedAppPage(page, 1);
+
+        const pageTitle = await viewAuthorizedAppPage.getPageTitle(page);
+        await expect(pageTitle).to.eq(viewAuthorizedAppPage.pageTitle(application.appName));
+      });
+      it('should check details', async function () {
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts:102]:
```suggestion

      it('should check details', async function () {
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        await dashboardPage.goToSubMenu(
+          page,
+          dashboardPage.advancedParametersLink,
+          dashboardPage.authorizationServerLink,
+        );
+
+        const pageTitle = await authorizationServerPage.getPageTitle(page);
+        await expect(pageTitle).to.eq(authorizationServerPage.pageTitle);
+      });
+      it('should delete authorized app', async function () {
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts]:
```suggestion
      });

      it('should delete authorized app', async function () {
```

**Review by boubkerbribri — COMMENTED** [review]
some spaces but LGTM

File: tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts (hunk 2 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts]: Fixed

File: tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/12_authorizationServer/02_viewDetail.ts:102]: Fixed

**Review by boubkerbribri — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**